examples/integration_test.py

```diff
@@ -331,6 +331,115 @@ def verify_avro():
         c.close()
 
 
+def verify_avro_https(*args):
+    from confluent_kafka import avro
+    avsc_dir = os.path.join(os.path.dirname(__file__), os.pardir, 'tests', 'avro')
+
+    # Avoid potential for never ending consume/produce cycle
+    timeout = time.time() + 15
+
+    # Producer config
+    conf = {'bootstrap.servers': bootstrap_servers,
+            'error_cb': error_cb,
+            'api.version.request': api_version_request,
+            'default.topic.config': {'produce.offset.report': True}}
+
+    # Create producer
+    if schema_registry_url:
+        conf['schema.registry.url'] = schema_registry_url
+        conf['schema.registry.ssl.ca.path'] = args[0]
+        conf['schema.registry.ssl.certificate.location'] = args[1]
+        conf['schema.registry.key.location'] = args[2]
+    else:
+        raise ValueError("Schema Registry URL required for the avro-https test")
```

Comment:
Reverse the logic to avoid nesting, and use `is None` checks:

`if schema_registry_url is None: \n raise..`